Extension/popup.js

```diff
@@ -6,35 +6,43 @@ function onWindowLoad() {
     }
   });
 
+  function showKeybinding(command) {
+    if (command.shortcut) {
+      document.getElementById(command.name).innerHTML += ' (' + command.shortcut + ')';
+    }
+  }
+
+  var commands = chrome.commands.getAll(function(commands) {

```

Comment:
It is necessary to set this to 'var commands'?  I don't see you using commands anywhere, but maybe this is some javascript foo I don't know.